### ISSUE TITLE
[BUG] QTreeWidget editor is too small

### ISSUE THREAD

**Issue by felipefarinon** [Alexhuszagh/BreezeStyleSheets#69]
## Description

The editor for QTreeWidgets is too small and conceals half of the text it is editing.

![small](https://user-images.githubusercontent.com/17036813/202720306-d4846d42-d87c-4951-aa9b-619fdcb9e12e.png)

## Prerequisites

- Qt version: 6.4
- Extension: None
- Theme: Dark and Light

**Comment by felipefarinon** [comment]
Sure. I reproduce the issue on Windows 10 Pro.

**Comment by Alexhuszagh** [comment]
I've fixed this by removing the padding from the QLineEdit in the widgets in #72. It should now look much better, and I've posted 2 examples below (both with Qt6, but they're identical between Qt5 and Qt6). Thanks for reporting, this was a very important fix.

![2022-11-18 17_39_57-Unittest for editable_list](https://user-images.githubusercontent.com/9440903/202821176-c94cc03a-650f-4537-af65-f982eec48833.png)
![2022-11-18 17_40_08-Unittest for editable_tree](https://user-images.githubusercontent.com/9440903/202821177-bebaaf70-f1e2-4f83-aa84-c427507af141.png)
